src/index.html

```diff
@@ -15,18 +15,13 @@ <h2 class="recent">Recent scores</h2>
             <a href="#" class="refresh">Refresh</a>
           </div>
           <ul class="high-list">
-            <li class="high-score">John : 100</li>
-            <li class="high-score">Jane : 95</li>
-            <li class="high-score">Luke : 90</li>
-            <li class="high-score">Ninja boy : 87</li>
-            <li class="high-score">Sarah : 85</li>
           </ul>
         </div>
         <div class="add-score">
           <h2 class="add">Add score</h2>
           <form action="#">
             <input type="text" class="name" placeholder="Your name" />
```

Comment:
- [ ] [Optional] You may consider using a ```require``` attribute to prevent submitting the form when the inputs are empty and give an error message to the user for a better UX experience